src/v2/view-builder/views/TerminalView.js

```diff
@@ -17,7 +17,7 @@ const Body = BaseForm.extend({
       messagesObjs.value
         .forEach(messagesObj => {
           const msg = messagesObj.message;
-          if (messagesObj.class && messagesObj.class === 'ERROR') {
+          if (messagesObj.class === 'ERROR') {
```

Comment:
@haishengwu-okta - not sure what this was supposed to be. `(messagesObj.class && messagesObj.class === 'ERROR')` is same as `(messagesObj.class === 'ERROR')`

Reply:
@anipendakur-okta i think it checks if `class` as property exists or not.. In some api responses we might not have `class` property

Reply:
If class didn't exist, `=== 'ERROR'` wouldn't be true hence would result in false. An existential check before it seems unnecessary.

Reply:
wouldn't it throw error `can't access class of undefiend`

Reply:
`messageObj` is an array element, so it would only fail if we had something like `[undefined]` which would not happen in this case. The change is still ok if we had something like `[{}]` (no `class` in the object).